### PR TITLE
Ft registration 181414716

### PR DESCRIPTION
**Pull request by Robertishimwe**
### What does this PR do?
Users should be able Signup/Register to Barefoot Nomad
### Description of Task to be completed?
**GIVEN** user successfully signs up/registers.
**WHEN** user passes appropriate signup parameters 
**THEN** user should be able to sign up.

### How should this be manually tested?
Get started By Running commands bellow:
  - clone repo
  - switch to [ft-registration-181414716](https://github.com/atlp-rwanda/code-ninjas-fe/tree/ft-registration-181414716) branch
  - Run `yarn install`: to install all dependencies
  - Run `yarn dev`: to start the application

`

Routes:

`

GET
[/registration](https)
REGISTRATION PAGE  `Register`

`

### What are the relevant pivotal tracker stories?
[#181414716](https://www.pivotaltracker.com/story/show/181414716)
### Questions:

### CODE REVIEW

**Review by sndicunguye — DISMISSED** [review]

**Review by itisWasp — DISMISSED** [review]

File: src/components/registration/formInPuts.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+//   return (
+//     <div>
+//       <label>{label}</label> <br />
+//       <input {...inputProps} onChange={onChange} />
+//       <p className="errorMassage">{errorMessage}</p>
+//     </div>
+//   );
+// }
+
+// export default FormInPuts;
```
Comment (ericus123) [inline src/components/registration/formInPuts.jsx]: what's the use of this???

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: src/components/registration/formInPuts.jsx (hunk 1 above)
Comment (Robertishimwe) [reply src/components/registration/formInPuts.jsx]: resolved

File: src/styles/verify.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.verifyRequest {
+    color: #ffff;
+    background-color: #1d97a7;
```
Comment (ericus123) [inline src/styles/verify.scss]: colors can't be hardcoded as well

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: src/components/login/google.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Google() {
+  return <div>Goooooo</div>;
+}
+
+export default Google;
```
Comment (ericus123) [inline src/components/login/google.jsx]: @Robertishimwe  what's the use of this component?

File: src/components/registration/registrationForm.jsx [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+  } = props;
+
+  return (
+    <div className={classes.container}>
+      <form onSubmit={handleSubmit} data-testid="regForm">
+        <Card className={classes.card}>
+          <Typography
+            data-testid="title"
+            style={{
+              color: '#1d97a7',
```
Comment (ericus123) [inline src/components/registration/registrationForm.jsx]: colors should not be hardcoded

File: src/components/registration/registrationForm.jsx [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+          </CardContent>
+          <CardActions className={classes.actions}>
+            <Button
+              data-testid="submit"
+              type="submit"
+              variant="contained"
+              color="primary"
+              disabled={isSubmitting}
+              style={{
+                backgroundColor: '#1d97a7',
```
Comment (ericus123) [inline src/components/registration/registrationForm.jsx]: colors should not be hardcoded.
fix this wherever you have them hardcoded

File: src/styles/footer.scss [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+.footer {
+    background-color: $primaryColor;
+    color: $secondaryColor;
+    padding: 5px 0px;
+    text-align: center;
+    font-family: $text, sans-serif;
+    font-weight: bolder;
+    // z-index: 9;
+    // position: fixed;
+    // bottom: 0;
```
Comment (ericus123) [inline src/styles/footer.scss]: remove this if not being used

File: src/styles/variable.scss [hunk 7]
```diff
@@ -0,0 +1,7 @@
+$primaryColor: #1d97a7;
+$secondaryColor: #ffffff;
+$tertiaryColor: #0000;
+$errorColor: #ff0000;
+$text: Montserrat;
```
Comment (ericus123) [inline src/styles/variable.scss:5]: are we using this font anywhere? this font-family is not the one used on figma

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: src/styles/variable.scss (hunk 7 above)
Comment (Robertishimwe) [reply src/styles/variable.scss:5]: it is the one we have
mmme

**Review by niyojuste — COMMENTED** [review]
@Robertishimwe Can you add an option for users to view their passwords as they input it

File: src/components/registration/registrationForm.jsx [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+              error={touched.password && Boolean(errors.password)}
+              margin="dense"
+              variant="outlined"
+              fullWidth
+            />
+            <FormControlLabel
+              control={<Checkbox defaultChecked />}
+              label="Show password"
+              onClick={showPassword}
+            />
```
Comment (niyojuste) [inline src/components/registration/registrationForm.jsx]: Nicely done! 
One small detail though, the logic is inverted. You can either start with:
- `Checkbox defaultChecked` in **FormControlLabel** with `type="text"` in **TextField** _or_
- `Checkbox` only in **FormControlLabel** with `type="password"` in **TextField**

**Review by niyojuste — DISMISSED** [review]

**Review by itisWasp — DISMISSED** [review]

**Review by ericus123 — APPROVED** [review]

**Pull request merged**